app/test_main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+from typing import Callable
+from unittest import mock
+from app.main import can_access_google_page
+
+import pytest
```

Comment:
fix this, according to best practicies 
https://peps.python.org/pep-0008/#:~:text=Imports%20are%20always,group%20of%20imports.